index.html

```diff
@@ -145,22 +122,25 @@ <h1 class = title>MATH WORLDS</h1>
 	<a href = https://www.google.com>What is special about MW games?</a>
 	</div>
 	</p>
-	
-	<img class = logo, src = "file:///C:/Users/The%20Critic/Downloads/Logo-Final-outlines.svg", width = 250px, height = 300vw, alt = "Happy Robot">
 <-->
 <div class="tab">
-  <button class="tablinks" onclick="openCity(event, 'Home')">Home</button>
-  <button class="tablinks" onclick="openCity(event, 'AboutUs')">About Us</button>
-  <button class="tablinks" onclick="openCity(event, 'MWGames')">MW Games</button>
+  <button class="tablinks" onclick="openCity(event, 'Home')"><strong>Home</strong></button>
```

Comment:
please do not use presentation tags like `<strong>`; style button labels using CSS.

Reply:
Will fix in the next pull request im'h.

Reply:
Done. Why is this? Purely convention?

Reply:
Hmm... "done" in what sence? Please DO NOT USE PRESENTATION TAGS like `strong`; style with CSS.

> Why is this?

To separate structure (HTML) from presentation (CSS).
This is also why CSS needs to be in a separate file.

Reply:
I did this, but was it in the wrong branch?